packages/atlas/src/components/_playlist/PlaylistListItem/PlaylistListItem.tsx

```diff
@@ -0,0 +1,120 @@
+import { useState } from 'react'
+import { CSSTransition } from 'react-transition-group'
+
+import { SvgActionEdit, SvgActionHide, SvgActionLinkUrl, SvgActionPlay, SvgActionShow } from '@/assets/icons'
+import { Text } from '@/components/Text'
+import { Tooltip } from '@/components/Tooltip'
+import { Button } from '@/components/_buttons/Button'
+import { ColumnGapBlock, RowGapBlock } from '@/components/_layouts'
+import { VideoThumbnail } from '@/components/_video/VideoThumbnail'
+import { useMediaMatch } from '@/hooks/useMediaMatch'
+import { cVar } from '@/styles'
+
+import {
+  Cell,
+  HoverContainer,
+  PlaylistDescription,
+  StyledPill,
+  StyledSvgActionTrash,
+  Wrapper,
+} from './PlaylistListItem.styles'
+
+export const PlaylistListItem = () => {
+  const mdMatch = useMediaMatch('md')
+  const [isHovered, setIsHovered] = useState(false)
+
+  const actionButtons = (
+    <ColumnGapBlock gap={1}>
+      <Tooltip text="Edit playlist" placement="top">
+        <Button icon={<SvgActionEdit />} variant="tertiary" />
+      </Tooltip>
+      <Tooltip text="Cody link" placement="top">
+        <Button icon={<SvgActionLinkUrl />} variant="tertiary" />
+      </Tooltip>
+      <Tooltip text="Play in Joystream" placement="top">
```

Comment:
I don't think this should be `Play in Joystream`, designs are pretty old. This should be `Play in ${appName}`